src/js/components/SelectMultiple/SelectMultipleContainer.js

```diff
@@ -365,7 +365,7 @@ const SelectMultipleContainer = forwardRef(
     if (help) {
       if (typeof help === 'string')
         helpContent = (
-          <Box flex={false} pad="xsmall">
+          <Box flex={false} pad={theme.selectMultiple?.helpText?.pad}>
```

Comment:
Is this a candidate for `helpText.container.pad`?

If in the future we ever wanted to allow theme to drive the size of the help text itself, we might want this to be the container?

Reply:
Good call, changed this to add container